app/models/answer.rb

```diff
@@ -26,17 +26,17 @@ def likers
 
   def answer_notify(current_user)
     # notify question author
-    if current_user.uid != self.author.uid
-      AnswerMailer.notify_question_author(self.author, self).deliver
+    if current_user.uid != self.node.author.uid
```

Comment:
Hi,

I don't know why line 30 is not showing up as a change. I changed it [from](https://github.com/publiclab/plots2/blob/master/app/models/answer.rb#L30)

`AnswerMailer.notify_question_author(self.author, self).deliver`

[to](https://github.com/Khamba/plots2/blob/master/app/models/answer.rb#L30)

`AnswerMailer.notify_question_author(self.node.author, self).deliver`